vault/resource_quota_lease_count.go

```diff
@@ -54,6 +54,12 @@ func quotaLeaseCountResource() *schema.Resource {
 				Optional:    true,
 				Description: "If set on a quota where path is set to an auth mount with a concept of roles (such as /auth/approle/), this will make the quota restrict login requests to that mount that are made with the specified role.",
 			},
+			"inheritable": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Computed:    true,
```

Comment:
Even though the Vault API returns a default of `false` when this value is not provided, we should not make this computed. With Computed true, if you add a test scenario that sets `inheritable` to true and then removes the field from the config you will get a failure.